src/main/kotlin/no/nav/syfo/melding/domain/MeldingTilBehandler.kt

```diff
@@ -84,28 +84,45 @@ private fun MeldingTilBehandler.getBrevKode(): BrevkodeType {
     }
 }
 
-fun MeldingTilBehandler.toJournalpostRequest(pdf: ByteArray) = JournalpostRequest(
-    avsenderMottaker = createAvsenderMottaker(behandlerPersonIdent, behandlerNavn),
-    tittel = "Dialogmelding til behandler",
-    bruker = Bruker.create(
-        id = arbeidstakerPersonIdent.value,
-        idType = BrukerIdType.PERSON_IDENT,
-    ),
-    dokumenter = listOf(
-        Dokument.create(
-            brevkode = this.getBrevKode(),
-            tittel = "Dialogmelding til behandler",
-            dokumentvarianter = listOf(
-                Dokumentvariant.create(
-                    filnavn = "Dialogmelding til behandler",
-                    filtype = FiltypeType.PDFA,
-                    fysiskDokument = pdf,
-                    variantformat = VariantformatType.ARKIV,
-                )
-            ),
-        )
-    ),
-)
+fun MeldingTilBehandler.toJournalpostRequest(pdf: ByteArray, isPaminnelse: Boolean) =
+    JournalpostRequest(
+        avsenderMottaker = createAvsenderMottaker(behandlerPersonIdent, behandlerNavn),
+        tittel = createTittel(isPaminnelse),
+        bruker = Bruker.create(
+            id = arbeidstakerPersonIdent.value,
+            idType = BrukerIdType.PERSON_IDENT,
+        ),
+        dokumenter = listOf(
+            Dokument.create(
+                brevkode = this.getBrevKode(),
+                tittel = createTittel(isPaminnelse),
+                dokumentvarianter = listOf(
+                    Dokumentvariant.create(
+                        filnavn = createTittel(isPaminnelse),
```

Comment:
Dette blir jo mest småpirk, men vi kunne laget `val tittel = createTittel(isPaminnelse)` og så gjenbrukt den i stedet for å kalle `createTittel` tre ganger. Det samme kan gjøres i `journalpostRequestGenerator()`-funksjonen.

Det har jo ikke så mye å si, så du kan bestemme. I så fall må funksjonene gjøres om til å ha `return`-statement og litt sånt, da.

Reply:
Skal se på det 👍🏻